src/graphics/program-lib/programs/standard.js

```diff
@@ -332,6 +332,13 @@ const standard = {
             code += chunks.baseNineSlicedTiledPS;
         }
 
+        // declare textureBias
+        if (options.nineSlicedMode === SPRITE_RENDERMODE_TILED) {
+            code += `const float textureBias = -1000.0;\n`;
+        } else {
+            code += `uniform float textureBias;\n`;
```

Comment:
_getUvSourceExpression can get called multiple times for a shader - will we have multiple uniform declarations?

Reply:
no, this is `_fsAddBaseCode` function.

Reply:
ah right, I missed that. all good then.